k8s-external-dns/_variables.tf

```
@@ -20,7 +20,13 @@ variable "namespace" {
 
 variable "domain_filters" {
   type        = string
-  description = "a comman-separated list of domains to allow"
+  description = "DEPRECATED: a comman-separated list of domains to allow"
+}
+
+variable "domain_filter_list" {
+  type        = list(any)
```

```suggestion
  type        = list(string)
```
Probably should be a list of strings here